action/action-util/src/main/java/com/powsybl/action/util/Scalable.java

```diff
@@ -38,6 +39,14 @@ static GeneratorScalable gen(String id) {
         return new GeneratorScalable(id);
     }
 
+    static Scalable getScalable(String id) {
```

Comment:
Even if it doesn’t fit with java coding style you should probably rename this method to scalable instead of getScalable to simplify the syntax for the end-user. Is groovy automatically bind scalable to getScalable ? Can you test it ? If it’s ok, keep getScalable

Reply:
Tested. It does not binding. It looks reasonable because it is method, not a field(attribute) of class.